src/plugins/cams/client.py

```
@@ -0,0 +1,236 @@
+"""CDS API client wrapper for CAMS data access."""
+
+import logging
+import os
+from pathlib import Path
+from typing import Dict, Any, Optional, List
```

Ruff (0.12.2)

`typing.Dict` is deprecated, use `dict` instead

(UP035)